cmake/projects/OpenSSL/hunter.cmake

```diff
@@ -10,6 +10,28 @@ include(hunter_cmake_args)
 include(hunter_download)
 include(hunter_pick_scheme)
 
+hunter_add_version(
```

Comment:
please add new versions at the bottom of the file, some diff programs are otherwise confused

Reply:
Fixed!